docs/source/concepts/generating-index.rst

```diff
@@ -126,20 +126,62 @@ Example: Building a channel
 
 To build a local channel and put a package in it, follow the directions below.
 
-#. Make the channel structure.
+#. Make the channel directory.
 
     .. code-block:: bash
 
       $ mkdir local-channel
       $ cd local-channel
-      $ mkdir linux-64 osx-64
 
-#. Put your favorite package in the channel.
+#. Install ``wget`` if you don't already have it.
+
+    Check if you have it.
+
+    .. code-block:: bash
+
+      $ which wget
```

Comment:
"which" is only a unix command, so this method is only available to mac and Linux users. 

I believe `where.exe wget` would be the Windows equivalent.

Reply:
Thanks Katherine.  I had assumed that any Windows users will be using Windows Subsystem for Linux. I'll add this.

Reply:
Updated. But this is dangerous as I don't understand windows, and don't have access to a windows machine.

Reply:
I have a vanilla Windows VM, so I attempted some testing. A few things:

- I can't get `conda install wget` to work. I had to pip install it.
- Once I did get it installed in an environment, `where wget`, `where.exe wget`, and `where wget.*` all give me the same thing: `INFO: Could not find files for the given pattern(s).`

So, seems like vanilla Windows is going to be a bit more complicated than everything Unix based.